Source/WebCore/page/FrameTree.cpp

```diff
@@ -126,7 +126,7 @@ static bool inScope(Frame& frame, TreeScope& scope)
 {
     auto* localFrame = dynamicDowncast<LocalFrame>(frame);
     if (!localFrame)
-        return false;
+        return true;
```

Comment:
Is it true that any remote frames passed in here would be inScope?

Reply:
I think so.  The scope seems to be used to prevent detached frames from being included.  I filed rdar://116268930 for future investigation.